### DISCUSSION TITLE
Click Event

### DISCUSSION THREAD

**Discussion by HCPSMW** [fluentribbon/Fluent.Ribbon#1001] [Q&A] [answered]
Hello, how can I set (in C# WPF fluent) an event for selecting a fluent:RibbonTabItem
![image](https://user-images.githubusercontent.com/96522302/148906836-c1cb06de-76f6-498e-b924-ee335c732786.png)

**And that's what I want:**
![image](https://user-images.githubusercontent.com/96522302/148907695-e875b790-4df1-45a1-8bf9-5b4242e9cfc4.png)

**Comment by HCPSMW** [comment, marked as answer]
```
<!-- ... -->
    <fluent:Ribbon Grid.Row="0" HorizontalAlignment="Stretch" SelectedTabChanged="Ribbon_SelectedTabChanged">
        <!-- ... -->
    </fluent:Ribbon>
<!-- ... -->
```
`````
//...
    private void Ribbon_SelectedTabChanged(object sender, SelectionChangedEventArgs e)
    {
        Console.WriteLine(((Fluent.Ribbon)sender).SelectedTabItem.Header);
    }
//...
`````

**Comment by batzen** [comment]
Hi,
you can either use the property `SelectedTabItem` on your `Ribbon` instance or set `IsSelected` to `true` on a specific `RibbonTabItem`.

Ah, misunderstood your question. Glad you found the event.

**Reply by HCPSMW** [reply]
**Thanks**
